# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/streams/UnderlyingSinkCallbackHelpers.h
# Commit: b8ecf23519c9
# Full Hash: b8ecf23519c9ab5180d08f00c712674bfe968dac
# Author: Yulia Startsev <ystartsev@mozilla.com>
# Date: 2022-02-21 21:39:46
# Regressor Bug: 1742437
# File Overlap Count: 5
# Description:
#   Bug 1742437 - re-scope all generic module/script loader files under js/loader; r=jonco
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D138133
# ==============================================================================

diff -r 85fd9fc9cffa -r b8ecf23519c9 dom/streams/UnderlyingSinkCallbackHelpers.h
--- a/dom/streams/UnderlyingSinkCallbackHelpers.h	Mon Feb 21 15:39:08 2022 +0000
+++ b/dom/streams/UnderlyingSinkCallbackHelpers.h	Mon Feb 21 15:39:09 2022 +0000
@@ -7,8 +7,8 @@
 #ifndef mozilla_dom_UnderlyingSinkCallbackHelpers_h
 #define mozilla_dom_UnderlyingSinkCallbackHelpers_h
 
+#include "js/loader/ModuleMapKey.h"
 #include "mozilla/HoldDropJSObjects.h"
-#include "mozilla/dom/ModuleMapKey.h"
 #include "mozilla/dom/Promise.h"
 #include "mozilla/dom/UnderlyingSinkBinding.h"
 #include "nsISupports.h"